lib/src/chat/chat.dart

```diff
@@ -279,15 +282,7 @@ class _ChatState extends State<Chat> with ChatComposer, ChatCreateMixin, InviteM
                       onTap: () => _chatTitleTapped(),
                       child: buildRow(imagePath, name, subTitle, color, context, isVerified),
                     ),
-              actions: <Widget>[
```

Comment:
Please keep this list here or create a sub widget https://iirokrankka.com/2018/12/11/splitting-widgets-to-methods-performance-antipattern/

I know we have this anti-pattern everywhere, but we try to reduce this over time :)

Reply:
I agree. Unfortunately I can't spot any method call here... 🤔

Reply:
I meant the getActions method found here https://github.com/open-xchange/ox-coi/commit/92231644b6504f1fe1c1269e5ca8004f2da8518d